modules/apps/portal-search/portal-search-test-util/src/main/java/com/liferay/portal/search/test/util/BaseSearchRequestBuilderHighlightTestCase.java

```diff
@@ -73,18 +74,23 @@ public void testSearchWithHighlight() throws Exception {
 			"alpha beta gamma alpha eta theta alpha zeta eta alpha iota",
 			"alpha beta gamma delta epsilon zeta eta theta iota alpha");
 
-		Highlight highlight = _highlightBuilderFactory.builder(
-		).addFieldConfig(
-			_fieldConfigBuilderFactory.builder(
-				getFieldName()
-			).build()
-		).fragmentSize(
-			20
-		).postTags(
-			"[/H]"
-		).preTags(
-			"[H]"
-		).build();
+		HighlightBuilder highlightBuilder = _highlightBuilderFactory.builder();
+
+		List<String> fieldNames = Arrays.asList(getFieldNames());
```

Comment:
Maybe Bchan would like to change the order 

            List<String> fieldNames = Arrays.asList(getFieldNames());

            HighlightBuilder highlightBuilder = _highlightBuilderFactory.builder();

Reply:
I agree, thanks for catching this and thanks for the kind words! :blush: